src/types/SdkEventType.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * String Enumerated list of SDK Event Types
+ * These events are transmitted from SDK -> ImageStudio
+ * This list is not exhaustive, and will grow over time
+ *
+ * Using String ENUMS reduces breaking changes in future,
+ * allowing for addition, deletion and re-ordering without consequence.
+ *
+ * These values are serialized/deserialized across post messages
+ * renaming will cause problems, and should be avoided.
+ * You are encouraged to migrate to a new ENUM to enforrce backwards comatability.
+ */
+export enum SDKEventType {
+  Focus = 'FOCUS', //data: {}
+  SDKMetadata = 'SDKMETADATA', //data: SDKMetadata
```

Comment:
Should this be underscore separated to be consistent with the other type values?

```suggestion
  SDKMetadata = 'SDK_META_DATA', //data: SDKMetadata
```

Reply:
Maybe, on reflection i suppose this does differ to the next `IMAGE_INPUT` - this would require changing of hte image studio PR, so for now its fine as it is.